CRAMv3.tex

```diff
@@ -732,9 +736,21 @@ \subsection{\textbf{Slice header block}}
 
 The slice header block is never compressed (block method=raw). For reference mapped 
 reads the slice header also defines the reference sequence context of the data 
-blocks associated with the slice. Mapped and unmapped reads can be stored within 
-the same slice similarly to BAM file. Slices with unsorted reads must not contain 
-any other types of reads.
+blocks associated with the slice. Mapped reads can be stored along with
```

Comment:
I'd propose including a table, similar to the one in https://github.com/samtools/hts-specs/issues/370, right in the spec. With the columns for the three Slice "reference context" types, and rows for every other header field affected by the reference context value (maybe with an additional row for `RI`), it would nicely summarize the whole reference context state space for Slice header.

Reply:
Thanks for adding the table. I'm not sure if this is a spec comment or a file format comment, but it seems like "Embedded Reference" is another Slice "type" that is mutually exclusive with the existing 3 types in the table. What should the refID value be for such a slice (or is that defined elsewhere?) Should it have it's own column in the table ?

Reply:
Also, while were at it, should this table (the one I suggested -;)) overlaps some with the table that immediately follows. Can/should they be merged ?

Reply:
Responded to the Embedded Reference question at #370.

I don't think the tables should be merged.  The main slice header fields table is conceptually similar to other header tables in the spec, and follows the common (type, name, description) pattern.  The new slice types summary table describes the logical interactions of the fields instead, so I see that as having a different purpose.

Reply:
Ok, well, I'm all in favor of continuing to tighten up this part of the spec, but I certainly don't have any objections to merging what we have so far, which looks good.

Reply:
Sorry I've been absent recently - illness.  I'll take a look at the start of next week.  Thanks for the updates.